src/functionalTest/java/uk/gov/hmcts/reform/pcs/functional/tests/SampleFunctionalTest.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.reform.pcs.functional.tests;
+
+import net.serenitybdd.junit5.SerenityJUnit5Extension;
+import net.serenitybdd.annotations.Steps;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import uk.gov.hmcts.reform.pcs.functional.steps.ApiSteps;
+
+@ExtendWith(SerenityJUnit5Extension.class)
+class SampleFunctionalTest {
+
+    private static final String BASE_URL = "https://jsonplaceholder.typicode.com";
```

Comment:
Why are we doing this ? can we not do a test of existing / root URL like the functional test that's deleted?